src/components/Menu/MenuItem/MenuButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './MenuItem.scss';
+
+type MenuButtonProps = {
+    children: React.ReactNode;
+    direction: 'left' | 'right';
+    index: number;
```

Comment:
You can drop the index prop if you simply use another css selector in the generator: `menu-item:nth-child(...)`

Reply:
Done ✅